frontend/src/APIClients/mutations/TestMutations.ts

```diff
@@ -0,0 +1,18 @@
+import { gql } from "@apollo/client";
+
+const PUBLISH_TEST_MUTATION = gql`
```

Comment:
To be consistent with the our mutation naming conventions:
```suggestion
const PUBLISH_TEST = gql`
```